packages/webdriverio/src/commands/browser/call.ts

```diff
@@ -3,19 +3,24 @@
  * is treated like a synchronous function. It accepts promises and stops the execution until
  * the promise has resolved.
  *
+ * This command helps to run asynchronous code within a synchronous context. With
+ * WebdriverIO depcrecating synchronous usage (see [RFC](https://github.com/webdriverio/webdriverio/discussions/6702))
+ * this command is not very useful anymore.
+ *
  * <example>
     :call.js
-    it('some testing here', () => {
-        browser.url('http://google.com')
+    it('some testing here', async () => {
```

Comment:
can this command be removed when everything is async anyway?

Reply:
That's what I would assume as well

Reply:
Yes and no 😄  as we continue to support sync until the end of Node.js v16 we can't get rid of the command just yet